### PR TITLE
Change default kaniko image to `gcr.io/k8s-skaffold/skaffold-helpers/busybox` from `busybox`

### PR DESCRIPTION
**Pull request by gsquared94**
Raising this PR to check integration tests with this change. Not intending to merge.

### CODE REVIEW

File: pkg/skaffold/constants/constants.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ const (
 
 	DefaultKustomizationPath = "."
 
-	DefaultBusyboxImage = "busybox"
+	DefaultBusyboxImage = "gcr.io/k8s-skaffold/busybox"
```
Comment (tejal29) [inline pkg/skaffold/constants/constants.go:42]: shd we follow the same naming convention as `skaffold-debug-support` and change this to `gcr.io/k8s-skaffold/skaffold-helpers/busybox`

Comment (gsquared94) [reply pkg/skaffold/constants/constants.go:42]: isn't it `DefaultDebugHelpersRegistry = "gcr.io/k8s-skaffold/skaffold-debug-support"` also?

**Review by tejal29 — APPROVED** [review]
Looks good to me. Can you specify in the description the steps to generate the image 
`gcr.io/k8s-skaffold/skaffold-helpers/busybox`

Thanks
Tejal

**Comment by tejal29** [comment]
Please update the PR description with steps you used to create the `gcr.io/k8s-skaffold/skaffold-helpers/busybox` image.

Thanks
Tejal

**Review by tejal29 — APPROVED** [review]

**Comment by gsquared94** [comment]
> Just took a look at the description. Looks like its not intended to merge.

apologies about the outdated description. Thanks for merging anyway.

> Please update the PR description with steps you used to create the gcr.io/k8s-skaffold/skaffold-helpers/busybox image.

done.

**Pull request merged**